src/NSwag.Annotations/OpenApiOperationAttribute.cs

```diff
@@ -35,5 +54,11 @@ public SwaggerOperationAttribute(string operationId)
 
         /// <summary>Gets or sets the operation ID.</summary>
         public string OperationId { get; private set; }
+
+        /// <summary>Gets or sets the operation summary.</summary>
+        public string Summary { get; protected set; }
```

Comment:
Why is this protected?

Reply:
SwaggerOperationAttribute is marked as Obsolete and descendant OpenApiOperationAttribute is recommended to use. However, (I suppose this is to preserve backward compatibility), SwaggerOperationAttribute still used to obtain OperationID property, and OpenApiOperationAttribute just "wraps" it.

In order to keep all properties "together", I've placed Summary and Description in SwaggerOperationAttribute, not in OpenApiOperationAttribute - but in order to force usage of last one, I've created constructors with Summary and Description in OpenApiOperationAttribute  and not in SwaggerOperationAttribute. This was the reason of having protected setter for those properties.

Probably it is a kind of "over-engeneering", and thosу properties should just be in OpenApiOperationAttribute.

Reply:
> I suppose this is to preserve backward compatibility

Exactly 

> Probably it is a kind of "over-engeneering", and thosу properties should just be in OpenApiOperationAttribute.

New functionality should be added only in OpenApiOperationAttribute i think, so fine for me. Can we use private instead of protected then?

Reply:
Yes, I think those properties should be in OpenApiOperationAttribute.
I moved them and made private setters - see new commit.